index.html

```diff
@@ -42,13 +44,218 @@ <h1>Organic</h1>
         </header>
     </div>
 
-
     
     <!---------How it works section----------->
 
-    
+  <section>
+       <div class="goods">
+            <h2>How it works?</h2>
+            <p>The principle of how markup works is as follows: 
+               You enter the address of a website or page into the address bar. 
+               The browser sends a request to this address and receives a file in HTML format. 
+               The code from the resulting file is sequentially converted into visual 
+               objects.</p>
+
+               
+               <div class="goods-box">
+                    <div class="box box1"></div>
+                    <div class="box box2">
+                     <img src="img/organic.png" style="width: 40px;height: 40px;">
+                     <p class="title">Organic Farming</p> 
+                     <p>Oh, and this is not a simple matter - farming. Either feed the pigs, 
+                        then milk the cows, then harness the horse... </p> 
+                    </div>
+                    <div class="box box3"></div>
+                    <div class="box box4"> 
+                        <img src="img/cow.png" style="width: 40px;height: 40px;">
+                        <p class="title">Animals husbandry</p>
+                        <p>An excellent family game that will not leave your family indifferent.</p>
+                    </div>
+                    <div class="box box5"></div>
+                    <div class="box box6">
+                        <img src="img/tractor.png" style="width: 40px;height: 40px;">
+                        <p class="title">Arable Farming</p>
+                        <p>Oh, and this is not a simple matter - farming. Either feed the pigs, 
+                            then milk the cows, then harness the horse... </p>
+                    </div>
+               </div>
+
+
+        
```

Comment:
Many empty lines are not allowed, usually no more that one empty line is acceptable